modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryCoordinatorFailureTest.java

```diff
@@ -273,8 +273,16 @@ public void awaitDrop() throws InterruptedException, IgniteCheckedException {
                 throw new IgniteCheckedException("Failed to wait for NodeAddFinishedMessage");
         }
 
-        /** {@inheritDoc} */
```

Comment:
Why replaced the inherited doc?

Reply:
fixed. i've reviewd code obe more time and i've veririden method instead of overloading. so doc is inherited now.
TcpDiscoveryConnectionCheckMessage is used as a stub instead of msg which was pased via parameter according to your comment:
// Replace logic routine message with a stub to update last-sent-time to avoid segmentation on
// connRecoveryTimeout.